flask/test/auth.py

```diff
@@ -19,33 +18,39 @@ def wrapper(*args):
 
     return wrapper
 
-
 def get_user_token():
     """Grabs a user token for Auth0
     Data structure as described here
     https://manage.auth0.com/dashboard/eu/boxtribute-dev/apis/5ef3760527b0da00215e6209/test"""  # line too long # noqa: E501
```

Comment:
this is probably where I get the AUTH0_CLIENT_TEST_ID and AUTH0_CLIENT_SECRET_TEST, right? 
and I have to select boxtribute-dev-api as API, right?

nice, that you figured out how to get the token at the end. I do not wanna know how much time it took.